stablehlo/dialect/StablehloOps.td

```diff
@@ -1028,16 +1028,16 @@ def StableHLO_OutfeedOp : StableHLO_Op<"outfeed",
 
     Example:
     ```mlir
-    %result = "stablehlo.outfeed"(%input0, %token) {
+    %result = "stablehlo.outfeed"(%inputs0, %token) {
       outfeed_config = ""
-    } : (tensor<3x3x3xi32>, !stablehlo.token) -> !stablehlo.token
+    } : (tensor<2x2x2xi64>, !stablehlo.token) -> !stablehlo.token
     ```
   }];
 
   let arguments = (ins
-    Variadic<HLO_Tensor>:$inputs,
-    HLO_Token:$token,
-    DefaultValuedStrAttr<StrAttr, "">:$outfeed_config
+    Variadic<HLO_Tensor>:$inputs, /*outfeed_i1*/
```

Comment:
I think we'd want to update constraints to say that all the inputs except last are tensors and the last input is a token? Something like:

```
(C1) `0 < size(results)`.
(C2) `is_tensor(results[:-1])`.
(C3) `is_token(results[-1])`.
```

Reply:
Since `outfeed` only returns a token, I've added (C3) in your proposal.

Reply:
Apologies, I was too hasty when writing the previous review and mixed up InfeedOp and OutfeedOp. Please disregard this comment.